# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1426696175 14400
#      Wed Mar 18 12:29:35 2015 -0400
# Node ID 749104c1bb1ba737acbc0d860559186b0d9df251
# Parent  210b301a3e397fa5867f65d267ddb7dcab54278d
* contributors.in: Add Johannes Leuschner to the list.

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -170,16 +170,17 @@ Miroslaw Kwasniak
 Rafael Laboissiere
 Kai Labusch
 Claude Lacoursiere
 Walter Landry
 Bill Lash
 Dirk Laurie
 Maurice LeBrun
 Friedrich Leisch
+Johannes Leuschner
 Thorsten Liebig
 Jyh-miin Lin
 Timo Lindfors
 Benjamin Lindner
 Ross Lippert
 David Livings
 Sebastien Loisel
 Erik de Castro Lopo
